src/astro/databases/postgres.py

```diff
@@ -24,5 +27,58 @@ def hook(self):
 
     @property
     def default_metadata(self) -> Metadata:
-        # TODO: comment why
-        return Metadata(database=self.hook.schema)
+        schema = self.hook.get_connection(self.conn_id).schema
+        return Metadata(database=schema)
```

Comment:
Is the database getting used anywhere for Postgres? Also, why do we match schema to db here? is it because of the Postgres hook?